api-client/src/main/java/com/xing/api/CallSpec.java

```diff
@@ -172,6 +172,13 @@
      */
     CallSpec<RT, ET> formField(String name, List<String> values);
 
+    /**
+     * Overrides the connection's default timeouts for this particular call.
+     *
+     * All timeouts in seconds.
```

Comment:
And I think the single comment already explains it pretty much, doesn't it? `Overrides the connection's default timeouts for this particular call.` - I wouldn't go down so far and explain the meaning of read or request timeout :)